frontend/src/components/SSEExample.js

```
@@ -0,0 +1,303 @@
+  };
+
+  const getConnectionStatusIcon = () => {
+    switch (connectionState) {
+      case 'OPEN':
+        return <Wifi className="h-5 w-5 text-green-500" />;
+      case 'CONNECTING':
+        return <RefreshCw className="h-5 w-5 text-yellow-500 animate-spin" />;
+      case 'CLOSED':
+      case 'ERROR':
+      default:
+        return <WifiOff className="h-5 w-5 text-red-500" />;
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)